### PR TITLE
Fix static analysis problems

### PR DESCRIPTION
**Pull request by szepeviktor**
I am a phpstan guy, you are psalm guys. 💥

### CODE REVIEW

File: src/Properties/ThemeProperties.php [hunk 1]
```diff
@@ -67,8 +67,7 @@ protected function __construct(string $themeDirectory)
         $properties = Properties::DEFAULT_PROPERTIES;
 
         foreach (self::HEADERS as $key => $themeKey) {
-            /** @psalm-suppress DocblockTypeContradiction */
-            $properties[$key] = $theme->get($themeKey) ?? '';
+            $properties[$key] = $theme->get($themeKey) ?: '';
```
Comment (szepeviktor) [inline src/Properties/ThemeProperties.php:70]: `WP_Theme::get` returns false on failure
https://developer.wordpress.org/reference/classes/wp_theme/get/

File: src/Package.php [hunk 2]
```diff
@@ -499,21 +499,19 @@ private function addModuleServices(Module $module, string $status): bool
                 break;
         }
 
-        if (!$services) {
+        if (!$services || !$addCallback) {
```
Comment (szepeviktor) [inline src/Package.php:502]: To aid static analysis.

File: src/Package.php [hunk 3]
```diff
@@ -209,7 +209,7 @@ class Package
 
     /**
      * @param Properties $properties
-     * @param ContainerInterface[] $containers
+     * @param ContainerInterface ...$containers
```
Comment (szepeviktor) [inline src/Package.php:212]: Variable-length argument lists is documented this way.

**Comment by szepeviktor** [comment]
I feel psalm will not understand these ...

**Comment by Chrico** [comment]
Hey @szepeviktor - we're using psalm. So if psalm does not understand the changes made for phpstan... mhm hard to merge 🙈 If you find a way to make it compat or fix it without breaking psalm: feel free to reopen 💪🏻

**Comment by szepeviktor** [comment]
All right @Chrico

**Pull request closed without merging**